actors/evm/tests/asm.rs

```diff
@@ -82,3 +83,19 @@ pub fn new_contract(name: &str, init: &str, body: &str) -> Result<Vec<u8>, etk_a
     contract_code.append(&mut body_code);
     Ok(contract_code)
 }
+
+// this is a hack to support mnemonics for the FEVM extension opcodes
+// it is really ugly, but the etk assmebler doesn't currently support any way to
+// directly embed (otherwise invalid) asm instructions in the stream... sigh.
+// Ideally we would just do them as macros like
+// %macro methodnum()
+//   %0xb1
+// %end
+// Note that to add insult to injury, macros cannot %include_hex... double sigh.
+// So fuck it, we'll just hack this until there is support.
+// See also https://github.com/quilt/etk/issues/110
+fn with_fevm_extensions(body: &str) -> String {
+    body.to_owned()
+        .replace("@callactor", "%include_hex(\"tests/opcodes/callactor.hex\")")
+        .replace("@methodnum", "%include_hex(\"tests/opcodes/methodnum.hex\")")
+}
```

Comment:
Nice work figuring out a workaround!

Reply:
Yeah, it was really annoying.... Not perfect, but we can live with it until upstream gets support for arbitrary instructions. If they don't act on my issue, I'll send them a pr.